src/mud/examples/adcirc.py

```diff
@@ -112,10 +112,10 @@ def tri_mesh_plot(
                 label=f"Recording Station {i}",
             )
     ax.legend()
-
-    save_figure(
-        f"si-{value}", save_path, close_fig=close_fig, dpi=dpi, bbox_inches="tight"
-    )
+    if save_path:
```

Comment:
Should be 
`if save_path is not None:`
See [PEP 8](https://peps.python.org/pep-0008/)

Reply:
hm, why doesn't flake8 complain about this?
my thinking was actually to check for both `None` and `""` simultaneously.
Though perhaps the behavior of `""` is fine... realized I'm not actually sure what would happen (would it try to save to root or to cwd?)

Reply:
> Also, beware of writing if x when you really mean if x is not None – e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

the type is `str`, and I did intend for `""` to evaluate in the same context (this is the only `str` that evals to False), so I was "careful" in the sense that this recommendation suggests. Question is... was I right about my assumption that `""` is a bad argument to pass to `save_figure`? I would prefer "cwd" to be given by `"."` which evals to True.

Reply:
Either works. I think PEP8 is just there to make the code more clear and readable in this case. "" evaluates to None but if not explicitly stated it just obfuscates the code a bit.

Reply:
fwiw, i ended up doing this.